dle/data/management/commands/load_hc_data.py

```diff
@@ -0,0 +1,423 @@
+                # logger.info(f"sleep 1s")
+                time.sleep(0.5)
+                drug_label_parsed += 1
+            # Click the next button
+            next_button = self.driver.find_element(by=By.ID, value="results_next")
+            if next_button is not None and drug_label_parsed < num_total_results:
+                next_button.click()
+                # Wait for a bit, take awhile for it to load
+                time.sleep(10)
+            else:
+                break
+
+        for url in self.error_urls.keys():
+            logger.warning(self.style.WARNING(f"error parsing url: {url}"))
+        logger.info(f"num_drug_labels_parsed: {self.num_drug_labels_parsed}")
+        logger.info(self.style.SUCCESS("process complete"))
+        return
+
+    def get_drug_label_from_row(self, row):
+        dl = DrugLabel()  # empty object to populate as we go
+        dl.source = "HC"
+
+        columns = row.find_all("td")
+        # DIN is located at the second column
+        # TODO: Is DIN same as source product number?
+        dl.source_product_number = columns[1].text.strip()
+
+        # Company is located at the third column
+        # TODO: Is Company same as marketer?
+        dl.marketer = columns[2].text.strip()
+
+        # product name is located at the fourth column
+        # strength is located at the 10th column
+        dl.product_name = columns[3].text.strip() + " " + columns[9].text.strip()
+
+        # The column under DIN is a clickable link to the drug details
+        link_to_drug_details = HC_BASE_URL + columns[1].find("a")["href"]
+        logger.info(f"Scraping URL {link_to_drug_details}")
+        response = self.requests_with_retries(link_to_drug_details)
+        if not response:
+            raise ValueError(f"unable to grab url contents {link_to_drug_details}")
+
+        soup = BeautifulSoup(response.text, "html.parser")
+        divs = soup.findAll("div", attrs={"class": "row"})
+        # Get the version date and the pdf link
+        dl.version_date = ""
+        dl.link = ""
+        for div in divs:
+            ps = div.findAll("p", attrs={"class": "col-sm-8"})
+            for p in ps:
+                if p.find("strong"):
+                    if "Date" in p.find("strong").text:
+                        spans = p.findAll("span")
+                        dl.version_date = spans[0].text
+                        dl.link = spans[1].find("a")["href"]
+        if dl.link == "":
+            logger.info(f"{dl.product_name} does not have a PDF label: {link_to_drug_details}")
+
+        # If version date is not available, then get the status date instead
+        if dl.version_date == "":
+            for div in divs:
+                p1 = div.find("p", attrs={"class": "col-sm-4"})
+                if p1 and p1.find("strong") and p1.find("strong").text == "Current status date:":
+                    p = div.find("p", attrs={"class": "col-sm-8"})
+                    dl.version_date = p.text
```

Comment:
How clean are the HC version dates? Do we need to use `dateparser`, or are they very standardized?

Reply:
The version dates are standardized. e.g., https://health-products.canada.ca/dpd-bdpp/info?lang=eng&code=96287